server/api/messageCenter/index.ts

```diff
@@ -0,0 +1,11 @@
+import { Express } from 'express'
+import { ApiEndPoint } from '@common/api/endpoint'
+import { getTopic } from './getTopic'
+import { addMessage } from './addMessage'
+
+export const MessageCenterApi = {
+  init: (express: Express): void => {
+    express.get(ApiEndPoint.MessageCenter.topic.get(), getTopic)
+    express.post(ApiEndPoint.MessageCenter.topic.addMessage(), addMessage)
```

Comment:
Do we need middleware for this @minotogna ?

Reply:
not now @sorja . this will be addressed in a later task